backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="oadp-rosa-backing-up-applications"]
+= Backing up applications ROSA clusters using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-rosa-backing-up-applications
+
+toc::[]
+
+You can use {oadp-first} with {product-rosa} (ROSA) clusters to back up and restore application data.
+
+ROSA is a fully-managed, turnkey application platform that allows you to deliver value to your customers by building and deploying applications.
+
+ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to speed up the building and delivering of differentiating experiences to your customers.
+
+You subscribe to the service directly from your AWS account.
+
+After you create your clusters, you can operate your clusters with the OpenShift web console or through link:https://docs.openshift.com/dedicated/ocm/ocm-overview.html[{cluster-manager-first}]. The ROSA service also uses OpenShift APIs and command-line interface (CLI) tools.
```

Comment:
```suggestion
After you create your clusters, you can operate your clusters with the OpenShift web console or through link:https://docs.openshift.com/dedicated/ocm/ocm-overview.html[{cluster-manager-first}]. You can also use ROSA service via OpenShift APIs and command-line interface (CLI) tools.
```

Reply:
@kaovilai 
I made the changes and suggestions you requested.
Do you have any other comments?